tests/acceptance/TestHelpers/Asserts/WebDav.php

```diff
@@ -64,7 +64,7 @@ public static function assertDavResponseElementIs(
 		self::assertEquals(
 			$expectedValue,
 			$result,
-			__METHOD__ . " " . $extraErrorText . "Expected '$expectedValue' in element $element got '$result'"
+			__METHOD__ . " " . $extraErrorText . "Expected '$expectedValue' in element $element got '$result'",
```

Comment:
do we need this type rules in linter 
```suggestion
			__METHOD__ . " " . $extraErrorText . "Expected '$expectedValue' in element $element got '$result'"
```